components/tabs/w-tab.vue

```diff
@@ -1,5 +1,59 @@
+<script setup>
+import { tab as ccTab } from '@warp-ds/component-classes';
+import { inject, computed, onBeforeUnmount } from 'vue';
+
+const props = defineProps({
+  label: String,
+  name: {
+    type: String,
+    required: true,
+  },
+});
+
+const controller = inject('tab-controller');
+const activeTab = inject('activeTab');
+
+const isActive = computed(() => props.name === activeTab.value);
+const setActive = () => (activeTab.value = props.name);
+
+controller.registerTab(props.name);
+onBeforeUnmount(() => {
+  controller?.unregisterTab?.(props.name);
+});
+
+const tabClasses = computed(() => ({
+  [ccTab.tab]: true,
+  ['active-tab']: isActive.value,
+  [ccTab.tabActive]: isActive.value,
+}));
+
+const iconClasses = computed(() => ({
+  [ccTab.icon]: true,
+  [ccTab.iconUnderlined]: true,
+  [isActive.value
+    ? ccTab.iconUnderlinedActive
+    : ccTab.iconUnderlinedInactive]: true,
+}));
+
+const contentClasses = computed(() => ({
+  [ccTab.contentUnderlined]: true,
+  [isActive.value
+    ? ccTab.contentUnderlinedActive
+    : ccTab.contentUnderlinedInactive]: true,
+}));
```

Comment:
```suggestion
const contentClasses = computed(() => [
  ccTab.contentUnderlined,
  isActive.value ? ccTab.contentUnderlinedActive  : ccTab.contentUnderlinedInactive],
]);
```
...perhaps this is a little bit easier to read? 😄 
(No a big fan of the `: true` in general 😅 )